README.md

```diff
@@ -153,21 +155,21 @@ This method looks for the "foo" variable.  If found, it is returned.  If not, th
 
 ### Reading Routes
 
-Routes on Platform.sh define how a project will handle incoming requests; that primarily means what application container will serve the request, but it also includes cache configuration, TLS settings, etc.  Routes may also have an optional ID, which is the preferred way to access them.
+[Routes](https://docs.platform.sh/configuration/routes.html) on Platform.sh define how a project will handle incoming requests; that primarily means what application container will serve the request, but it also includes cache configuration, TLS settings, etc.  Routes may also have an optional ID, which is the preferred way to access them.
 
 ```go
 config.Route("main")
 ```
 
-The `Route()` method takes a single string for the route id ("main" in this case) and returns the corresponding Route struct.  Its second return is a boolean indicating if the route was found.  It is best used like so:
+The `Route()` method takes a single string for the route ID ("main" in this case) and returns the corresponding `route` struct.  Its second return is a boolean indicating if the route was found.  It is best used like so:
```

Comment:
Route should be capitalized since it's the proper name of a public struct, and thus capitalized.  (It's public-scoped, which in Go means it must have a capital first letter.)

Reply:
Gravy